### ISSUE TITLE
get_from_cab fails when path has whitespaces

### ISSUE THREAD

**Issue by tuxtergames** [bottlesdevs/Bottles#1103]
**Describe the bug**
Fail to install games stores.

**Installation**
 - Method: fedora dnf command
 - Version 22.2.28 trento-1

**To Reproduce**
1. Go to inside my bottle 
2. Click on installers
3. Scroll down to epic or ubisotf store and try to install, i'll get a "d3dcompiler43" infinity information
4. See error through terminal output

```
> bottles 
(15:10:32) INFO Performing Bottles checks... 
(15:10:32) INFO Dxvks found:
         - dxvk-1.9.4
 
(15:10:32) INFO Vkd3ds found:
         - vkd3d-proton-2.5
 
(15:10:32) INFO Nvapis found:
         - dxvk-nvapi-v0.5.2
 
(15:10:32) INFO Runners found:
         - sys-wine-7.2
         - Proton-7.3-GE-1
         - caffe-7.2
         - lutris-ge-7.2-2
 
(15:10:32) INFO Bottles found:
         - Ubisoft-Connect
 
(15:10:32) INFO Bottles Started! 
(15:10:40) INFO Setting Key: [discrete_gpu] to [True] for bottle: [Ubisoft Connect]… 
(15:10:40) INFO Setting Key: [pulseaudio_latency] to [True] for bottle: [Ubisoft Connect]… 
(15:10:40) WARNING Cannot find states.yml file for bottle: [Ubisoft Connect] 

(bottles:17107): Handy-CRITICAL **: 15:10:40.233: hdy_stackable_box_set_visible_child_name: assertion 'contains_child' failed
(15:10:44) INFO Installing dependency [d3dcompiler_43] in bottle [Ubisoft Connect]. 
(15:10:45) WARNING File [directx_Jun2010_redist.exe] already exists in temp, skipping. 
Home/Ubisoft-Connect/drive_c/windows/syswow64/: No such file or directory
/home/tuxter/.local/share/bottles/temp/d3dcompiler_43_x86/Jun2010_D3DCompiler_43_x86.cab: WARNING; possible 5960 extra bytes at end of file.
(15:10:45) ERROR Error while running async job: <bound method InstallerManager.install of <bottles.backend.managers.installer.InstallerManager object at 0x7f8ca2ebabf0>>
        Exception: [Errno 2] Arquivo ou diretório inexistente: '/home/tuxter/Bottles Home/Ubisoft-Connect/drive_c/windows/syswow64//D3DCompiler_43.dll'
 
  File "/usr/local/share/bottles/bottles/utils.py", line 103, in __target
    result = self.task_func(*args, **kwargs)
  File "/usr/local/share/bottles/bottles/backend/managers/installer.py", line 351, in install
    res = self.__install_dependencies(_config, dependencies, widget)
  File "/usr/local/share/bottles/bottles/backend/managers/installer.py", line 123, in __install_dependencies
    res = self.__manager.dependency_manager.install(_config, _dep)
  File "/usr/local/share/bottles/bottles/backend/managers/dependency.py", line 145, in install
    res = self.__perform_steps(config, step)
  File "/usr/local/share/bottles/bottles/backend/managers/dependency.py", line 239, in __perform_steps
    if not self.__step_get_from_cab(config=config, step=step):
  File "/usr/local/share/bottles/bottles/backend/managers/dependency.py", line 492, in __step_get_from_cab
    shutil.move(
  File "/usr/lib64/python3.10/shutil.py", line 825, in move
    copy_function(src, real_dst)
  File "/usr/lib64/python3.10/shutil.py", line 434, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib64/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
```

**Expected behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/26095484/156225944-e48d6f42-12a8-4c31-9bdb-a7b8e3bceb78.png)

**Desktop (please complete the following information):**
 - OS: Fedora 35 nobara
 - Version  22.2.28 trento-1
 - Display server: Xorg or wayland
**Additional context**

**Comment by mirkobrombin** [comment]
I was able to reproduce setting the bottles path to `Bottles Home` in my homedir. I'll make a patch for the next release (2022.3.14). In the meantime you can workaround this removing the space in your path and setting Bottles to the new one.

Thanks for reporting.

**Comment by pietrodicaprio** [comment]
We found the issue and a fix has been applied. It's going to be shipped with the next release (2022.3.14)
